src/typescript/11-keyof-typeof/keyof-typeof.md

```diff
@@ -152,11 +212,14 @@ type CarnivoreKeys = keyof typeof carnivore;
 let carnivoreKey: CarnivoreKeys;
 
 carnivoreKey = "name";
-carnivoreKey = "Some value"; // ERROR: Type '"Some value"' is not assignable to type '"name" | "type" | "weightInKilograms"'
+carnivoreKey = "Some value";
 carnivoreKey = "type";
 ```
+@highlight 12
+
+Note that in the above example assigning `"Some value"` to `carnivoreKey` will generate an error: `ERROR: Type '"Some value"' is not assignable to type '"name" | "type" | "weightInKilograms"'`
 
-A strange, but the common occurrence of this is an enum (learn more about enums: [learn-typescript/types#enum]). Enums in TypeScript are types before the code is compiled and an object during execution. If we run into a situation where we want to get the keys of an enum, the only way to do so is to use `keyof` and `typeof` together
+A strange, but the common occurrence of this is an enum (learn more about enums: [learn-typescript/types#enum]). Enums in TypeScript are types before the code is compiled and an object during execution. If we run into a situation where we want to get the keys of an enum, the only way to do so is to use `keyof` and `typeof` together:
```

Comment:
A strange?

These “learn more about” links can be links on the words, they don’t need to be in parenthesis.